### DISCUSSION TITLE
API data collection from Allure TestOps in Prometheus format

### DISCUSSION THREAD

**Discussion by NekrasovAM** [allure-framework/allure2#2465] [TestOps]
Hi!
I want to collect metrics using API methods in the Prometheus format.
For example, project statistics. I want to receive this data in Prometheus in order to monitor changes in the project.
![project_stats](https://github.com/allure-framework/allure2/assets/3220426/6bb49dc3-7542-4272-9089-21c22d389633)

How can i do this?

**Comment by cheshi-mantu** [comment]
@NekrasovAM 
Hi Artyom
could you please create a support request at https://help.qameta.io? It's a support portal dedicated to Allure Testops related stuff.
We are going to need a bit more imputs from your side to provide sensible answer.

cheers
